test/parallel/test-repl-history-navigation.js

```diff
@@ -554,6 +555,18 @@ const tests = [
     expected: [],
     clean: false
   },
+  {
+    env: { NODE_REPL_HISTORY: defaultHistoryPath },
+    test: ['const util = {}', ENTER,
+           'proc', RIGHT, ENTER],
```

Comment:
@aduh95 When no using SIGINT the REPL is not closed, and when using ENTER more data then I need is outputted so it'll be unreadable in the `expected: []`. Currently the test does not pass since more data is outputted then what in the expected

Reply:
If you close the REPL, subsequent tests would fail. For now it doesn't matter because it's the last of the list, but that would be a problem next time we want to add one to the list. Also the other test cases do not use `SIGINT`, it's probably a good idea to keep the same pattern.

Reply:
Let's reuse `util`, so we know to expect an empty object.
```suggestion
           'ut', RIGHT, ENTER],
```